CHANGELOG-7.0.md

```diff
@@ -0,0 +1,5 @@
+# Changes from 6.0 to 7.0
+
+See [breaking changes](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/breaking-changes-7.0.html).
+
+## ???
```

Comment:
Not sure what to put here really.

Reply:
I’ll probably have the chance to look at this on Friday afternoon. Looking forward to it.

Reply:
i used this file to add additional comments to the list of breaking changes that are specific to this library, e.g. changes in the Go API.